frontend/src/components/page-connect/map-details.js

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { StaticImage } from "gatsby-plugin-image";
+import { SecondaryButtonLink } from "../Button";
+
+const MapDetails = () => {
+  return (
+    <div className="flex flex-col items-center justify-center pt-5 bg-Neutral-200 md:flex-row md:h-[30.25rem] md:px-[8.125rem] md:flex-shrink-0 md:bg-[#fff]">
+      <div className=" h-[12.6875] px-[3.66rem] gap-5 md:h-[26.25rem] md:rounded-l-[1.25rem] md:w-[23.75rem] md:px-25  md:bg-Neutral-200 ">
```

Comment:
also here you should not need to set height. allow css to state what the height of the container.